cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -437,10 +440,22 @@ export class CanvasViewImpl implements CanvasView, Listener {
                     .filter((_state: any): boolean => (
                         _state.clientID === self.activeElement.clientID
                     ));
-                if (['cuboid', 'rectangle'].includes(state.shapeType)) {
+                if (state.shapeType === 'rectangle') {
                     e.preventDefault();
                     return;
                 }
+                if (state.shapeType === 'cuboid') {
+                    if (e.shiftKey) {
+                        const points = translate(pointsToArray((e.target as any)
+                            .parentElement.parentElement.instance.attr('points')));
+                        self.onEditDone(
+                            state,
+                            points,
+                        )
+                        e.preventDefault();
+                        return;
+                    }
+                }
```

Comment:
Is this to reset perspective?

Reply:
@bsekachev, yes

Reply:
@ActiveChooN 

Two different event listener are responsible for the same feature. It makes me confused